test/rai/pipeline_boston_analyse.yaml

```diff
@@ -25,14 +25,14 @@ jobs:
       categorical_features: '[]'
       continuous_features: '["CRIM", "ZN", "INDUS", "CHAS", "NOX", "RM", "AGE","DIS", "RAD", "TAX", "PTRATIO", "B", "LSTAT"]'
 
-  register-model-job:
+  register_model_job:
     type: command
     component: azureml:register_model:VERSION_REPLACEMENT_STRING
     inputs:
       model_input_path: ${{parent.jobs.train-model-job.outputs.model_output}}
```

Comment:
Don't the names need updating in these sections too?

Reply:
Good catch, they do need to be updated.